docs/tools/index.rst

```diff
@@ -15,3 +15,4 @@ Develop Tools
     debugging
     community
     developer
+    spark_block_cleaner
```

Comment:
I'd like add a new entrance like `tools`, then we can put this into `tools`

Reply:
Good suggestion, i will move this.